inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantServiceImpl.java

```diff
@@ -45,6 +45,9 @@ public class InlongTenantServiceImpl implements InlongTenantService {
     @Override
     public InlongTenantInfo get(String name) {
         InlongTenantEntity entity = inlongTenantEntityMapper.selectByName(name);
+        if (entity == null) {
+            return null;
```

Comment:
Do we need some warn or error logs here?

Reply:
fixed, thx